test/e2e/framework/framework.go

```diff
@@ -124,6 +124,7 @@ func (f *Framework) WaitForGameServerState(gs *v1alpha1.GameServer, state v1alph
 }
 
 // WaitForFleetCondition waits for the Fleet to be in a specific condition or fails the test if the condition can't be met in 5 minutes.
+// nolint: dupl
```

Comment:
Curious - why is there `nolint: dupl` here?

Reply:
We get two methods which are similar WaitForFleetCondition and WaitForFleetAutoscalerCondition. So `make tests` fails without it. You can see the output in the BuildFailed above.